deploy/test/global-manifests.yaml

```diff
@@ -21,4 +21,4 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: devconsole
-  namespace: devconsole-e2e-test
\ No newline at end of file
+  namespace: devconsole-e2e-test
```

Comment:
I think we need to update this global-manifests.yaml with updated cluster role

Reply:
yes but that would be another PR for now i focus on manual deployment
odc-132 is to deal with e2e test

Reply:
sure